client/src/pages/platform/workflow-editor/components/properties/components/PropertyComboBox.tsx

```diff
@@ -334,7 +334,7 @@ const PropertyComboBox = ({
                 </div>
             )}
 
-            <Popover onOpenChange={setOpen} open={open}>
+            <Popover modal={true} onOpenChange={setOpen} open={open}>
```

Comment:
just `modal` should be enough (boolean props default to `false` if absent, if present they're set to `true`, no need to specify `true`)